packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -443,6 +447,10 @@ export default {
 
         formattedMobileNumberForScreenReader () {
             return this.customer.mobileNumber ? [...this.customer.mobileNumber].join(' ') : '';
+        },
+
+        getCheckoutError () {
```

Comment:
It should probably be called `getCheckoutErrorType` given you're using it as a type in the template.